hera_qm/xrfi.py

```diff
@@ -1554,7 +1557,9 @@ def xrfi_run_step(uv_files=None, uv=None, uvf_apriori=None,
                 # No partial i/o for uvcal yet.
                 uv.read_calfits(uv_files)
                 if a_priori_flag_yaml is not None:
-                    uv = qm_utils.apply_yaml_flags(uv, a_priori_flag_yaml)
+                    uv = qm_utils.apply_yaml_flags(uv, a_priori_flag_yaml,
```

Comment:
Is there a reason this only gets applied to uvcal objects, but not uvdata objects?

Reply:
It also gets applied to uvdata objects but because these lines only read the metadata, it doesn't happen until here 1601 when the data is actually read in.